EpiAware/src/EpiLatentModels/models/AR.jl

```diff
@@ -81,23 +81,26 @@ Generate a latent AR series.
     damp_AR ~ latent_model.damp_prior
     ϵ_t ~ filldist(Normal(), n - p)
 
-    ar = accumulate_scan(ARStep(damp_AR), ar_init, σ_AR * ϵ_t)
+    ar_step = ARStep(reverse(damp_AR))
```

Comment:
For efficieny I don't think we want to do this at run time?

Reply:
To avoid doing `reverse`? That would suggest reversing at the constructor level I think.

Reply:
e.g. we add a `rev_damp_prior` field to `AR` and have `damp_AR ~ latent_model.rev_damp_prior`. Obvs we could just reverse `damp_prior` but that might get confusing for users?